exercises/eliuds-eggs/canonical-data.json

```diff
@@ -0,0 +1,50 @@
+{
+  "exercise": "eliuds-eggs",
```

Comment:
I think all the practice exercises have more direct names which map to the problem being solved, like pop count. I may be mistaken but I think the cute names are only used in concept exercises.

Reply:
Exercise renamed to Pop Count